src/jit/gentree.h

```diff
@@ -1476,19 +1477,44 @@ struct GenTree
     }
 
     static inline bool RequiresNonNullOp2(genTreeOps oper);
-    bool IsListForMultiRegArg();
+    bool IsValidCallArgument();
 #endif // DEBUG
 
     inline bool IsFPZero();
     inline bool IsIntegralConst(ssize_t constVal);
 
     inline bool IsBoxedValue();
 
-    bool IsList() const
+    static bool OperIsList(genTreeOps gtOper)
     {
         return gtOper == GT_LIST;
     }
 
+    bool OperIsList() const
+    {
+        return OperIsList(gtOper);
+    }
+
+    static bool OperIsFieldList(genTreeOps gtOper)
+    {
+        return gtOper == GT_FIELD_LIST;
+    }
+
+    bool OperIsFieldList() const
+    {
+        return OperIsFieldList(gtOper);
+    }
+
+    static bool OperIsAnyList(genTreeOps gtOper)
```

Comment:
Also static

---
In reply to: [79510306](https://github.com/dotnet/coreclr/pull/7252#discussion_r79510306) [](ancestors = 79510306)